### PR TITLE
Frontier Template: PoV Handling

### PR DESCRIPTION
**Pull request by chexware**
This PR adds PoV configuration to the Frontier container chain template and relevant tests  for it.

TO DO:

- [x] fix: block proof size for all PoV tests seems to be 0 which makes all of them fail
- [x] fix: failing non-PoV tests
- [x] fix: failing PoV tests
- [x] add block proof size output to dancelight' node service config

### CODE REVIEW

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts [hunk 1]
```diff
@@ -50,8 +50,8 @@ describeSuite({
                 const { result, block } = await context.createBlock(rawSigned);
 
                 log(`block.proofSize: ${block.proofSize} (successful: ${result?.successful})`);
-                expect(block.proofSize).toBeGreaterThanOrEqual(15_000);
-                expect(block.proofSize).toBeLessThanOrEqual(25_000n + emptyBlockProofSize);
+                expect(block.proofSize).toBeGreaterThanOrEqual(30_000);
```
Comment (girazoki) [inline test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts:53]: do these values make sense? why are they different to moonbeam ones?

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    id: "DF1301",
+    title: "PoV controlled by gasLimit",
+    foundationMethods: "dev",
+    testCases: ({ context, it, log }) => {
+        let proxyAddress: `0x${string}`;
+        let proxyAbi: Abi;
+        let contracts: HeavyContract[];
+        let callData: `0x${string}`;
+        const MAX_CONTRACTS = 20;
+        const EXPECTED_POV_ROUGH = 38_000; // bytes
```
Comment (girazoki) [inline test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts:17]: same thing with this, why is our expected pov different than moonbeam one?

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts (hunk 1 above)
Comment (chexware) [reply test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts]: Our tests seems to fail with the original values. Reverted them in https://github.com/moondance-labs/tanssi/pull/716/commits/58db5bd6992983f6a2df28a7ee03b2569cc69a90 to new values and will be looking into resolving the issue now

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts (hunk 2 above)
Comment (chexware) [reply test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts]: Our tests seems to fail with the original values. Reverted them in https://github.com/moondance-labs/tanssi/pull/716/commits/58db5bd6992983f6a2df28a7ee03b2569cc69a90 to new values and will be looking into resolving the issue now

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts (hunk 1 above)
Comment (chexware) [reply test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts:53]: Reverted to Moonbeam's values before the update in https://github.com/moondance-labs/tanssi/pull/716/commits/e8dfc8d9c0c3b23570326143b38b5d72e3b2a742

File: test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts (hunk 2 above)
Comment (chexware) [reply test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts]: Reverted to Moonbeam's values before the recent Frontier update in https://github.com/moondance-labs/tanssi/pull/716/commits/e8dfc8d9c0c3b23570326143b38b5d72e3b2a742

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**